portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -14,78 +14,92 @@
 
 package com.google.sps.servlets;
 
-import com.google.appengine.api.datastore.DatastoreService;
-import com.google.appengine.api.datastore.DatastoreServiceFactory;
-import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.*;
 import com.google.gson.Gson;
+import com.google.sps.utils.CommentDataStore;
 import com.google.sps.data.Comment;
+import com.google.sps.utils.CommentUtils;
 
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-import java.time.LocalDateTime;
 import java.util.ArrayList;
 import java.util.List;
 
 /** Servlet that handles getting and posting comment content. */
 @WebServlet("/comment")
 public final class CommentServlet extends HttpServlet {
-  private final List<Comment> comments = new ArrayList<>();
+
+  static class Constants {
```

Comment:
Nit: since these var no longer used else where. We can just make it private static final string fields of this class.
E.g 
private final static final String PARAM_NAME_COMMENTER = "commenter";
private final static final String PARAM_NAME_CONTENT = "content";

Reply:
Fixed

Reply:
Maybe can also remove the nested class